tests/unit/test.py

```diff
@@ -30,6 +30,19 @@ def setUp(self):
         main.app.config["TESTING"] = True
         self.app = main.app.test_client()
 
+    def post(self, *args, **kwargs):
+        rv = self.app.post(*args, **kwargs)
+        return rv.status_code, json.loads(rv.data.decode())
+
+    def put(self, *args, **kwargs):
+        rv = self.app.put(*args, **kwargs)
+        return rv.status_code, json.loads(rv.data.decode())
+
+    def delete(self, *args, **kwargs):
+        rv = self.app.delete(*args, **kwargs)
+        # there should be no content on delete method
```

Comment:
This is strange, because fake_runbooks.py:74 returns returns "{}":

     flask.jsonify({}), 204